.github/workflows/build-processor.yml

```diff
@@ -0,0 +1,39 @@
+name: Build and release processor
+
+on:
+  push:
+    tags:
+      - "processor-*"
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.19
```

Comment:
Any specific reason to be using `v2` of the action, looks like `v5` is the current version? Upgrading provides some additional features like retrieving the go version from the mod file & dependency caching (not sure how this action will deal with a non-existant `go.sum`, might be better to add this once the `go.sum` exists).
```suggestion
      - name: Setup Go
        uses: actions/setup-go@v5
        with:
          go-version-file: 'script/go.mod'
          cache-dependency-path: 'script/go.sum'
```

Reply:
[Looks like the file not existing shouldn't be a problem](https://github.com/actions/setup-go?tab=readme-ov-file#caching-dependency-files-and-build-outputs):
> If some problem that prevents success caching happens then the action issues the warning in the log and continues the execution of the pipeline.

Reply:
Indeed, looks like it works with a simple warning.

![CleanShot 2024-03-13 at 07 37 02@2x](https://github.com/1Password/1password-teams-open-source/assets/6392049/e9225f55-cbbd-45b8-ae01-fc626f5bffb5)

Good thinking!

Reply:
Thanks for catching these outdated versions.